src/richie/apps/search/filter_definitions/courses.py

```diff
@@ -368,8 +385,16 @@ def aggs_include(self):
         """
         if self.reverse_id:
             if self.base_page:
-                node = self.base_page.node
-                return f".*-{node.path:s}.{{{node.steplen:d}}}"
+                if not self._aggs_include:
+                    self._aggs_include = []
+                    for page in self.base_page.get_child_pages().select_related("node"):
+                        path = page.node.path
```

Comment:
Since what you are looking for is on the node, not on the page, it's probably better to work on the node directly?
Maybe something like this:
```
paths = self.base_page.node.get_children().values_list("path", flat=True)
return [f"{prefix}-{path}" for path in paths for prefix in ["P", "L"]]
```